[codemod] Fix unused-value issue in caffe2/aten/src/ATen/cuda/detail/CUDAHooks.cpp +4

r-barnes opened the pull request:
Summary:
LLVM has a warning `-Wunused-value` which we treat as an error because it's so often diagnostic of a code issue. Unused values often indicate a programming mistake, but can also just be unnecessary cruft that harms readability and performance.

For questions/comments, contact r-barnes.

 - If you approve of this diff, please use the "Accept & Ship" button :-)

Test Plan: Sandcastle

Differential Revision: D69945678




cc @H-Huang @awgu @kwen2501 @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o

Skylion007 reviewed (approved):
LGTM once the missing includes are added.

eqy commented on torch/csrc/cuda/shared/nvtx.cpp:
```diff
@@ -42,8 +45,10 @@ static void* device_nvtxRangeStart(const char* msg, std::intptr_t stream) {
   handle->id = 0;
-  cudaLaunchHostFunc(
-      (cudaStream_t)stream, device_callback_range_start, (void*)handle);
+  TORCH_CHECK(
```
Is `C10_CUDA_CHECK` not preferred here for some reason?

r-barnes replied:
Switching to it.

r-barnes committed:
[codemod] Fix unused-value issue in caffe2/aten/src/ATen/cuda/detail/CUDAHooks.cpp +4 (#147555)

Summary:

LLVM has a warning `-Wunused-value` which we treat as an error because it's so often diagnostic of a code issue. Unused values often indicate a programming mistake, but can also just be unnecessary cruft that harms readability and performance.

For questions/comments, contact r-barnes.

 - If you approve of this diff, please use the "Accept & Ship" button :-)

Test Plan: Sandcastle

Differential Revision: D69945678

r-barnes commented:
@pytorchbot merge

r-barnes commented:
@pytorchbot merge -f "Formatting issue was not caused by my code"
